core/arch/arm/pta/benchmark.c

```diff
@@ -43,6 +43,21 @@
 
 struct tee_ts_global *bench_ts_global;
 static struct mutex bench_reg_mu = MUTEX_INITIALIZER;
+static uint32_t prev_cntkctl;
+
+static void timer_enable_el0_access(void)
```

Comment:
I'd prefer if this function returned the previous state, similar to for instance `thread_mask_exceptions()` and `thread_unmask_exceptions()`. The calling function would of course need to store the result in `prev_cntkctl`, but that would make it more clear that there's a global state involved.

Reply:
Please check my latest commit to be sure that I understood you correctly